src/temp/ImagesSlider/ImagesSlider.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement, useEffect } from 'react';
+import Stories from 'react-insta-stories';
+import getImageUrl from '../../utils/getImageURL';
+import styles from './Slider.module.scss';
+import { SliderImage } from '../../features/projects/components/PlantLocation/ImageSlider';
+
+interface Props {
+  images: SliderImage[];
+  height: number | string;
+  imageSize: string;
+  type: string;
+}
+
+const ImagesSlider = ({ images, height, imageSize, type }: Props) => {
+  const [slider, setSlider] = React.useState<ReactElement>();
+  const projectImages: { content: () => ReactElement }[] = [];
+
+  useEffect(() => {
+    images.forEach((sliderImage) => {
+      if (sliderImage.image) {
+        const imageURL = getImageUrl(type, imageSize, sliderImage.image);
+
+        projectImages.push({
+          content: () => (
+            <div
+              className={styles.sliderContent}
+              style={
+                type === 'coordinate'
+                  ? { background: `url(${imageURL})` }
+                  : {
+                      background: `linear-gradient(180deg, rgba(0, 0, 0, 0.00) 0%, rgba(0, 0, 0, 0.49) 100%),url(${imageURL})`,
+                    }
+              }
+            >
+              <p className={styles.sliderContentText}>
```

Comment:
In the useEffect hook, JSX elements are being pushed into an array (projectImages). While this approach can work, it can become difficult to read and maintain as the code grows. Consider breaking down complex JSX into separate components or functions for better readability.